File: ConfigurePcLaravel/configurePc-App/app/Models/Log.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class Log extends Model
{
    protected $fillable = [
        'timestamp',
        'description',
        'level_id',
        'source_id',
        'type_id',
    ];

    public function level()
    {
        return $this->belongsTo(Level::class, 'level_id', 'id');
    }

    public function source()
    {
        return $this->belongsTo(Source::class, 'source_id', 'id');
    }

    public function type()
    {
        return $this->belongsTo(Type::class, 'type_id', 'id');
    }
}

File: ProductionSystem/PersistencePlugin/Models/log.cs
using System;
using System.Collections.Generic;

namespace PersistencePlugin.Models;

public partial class log
{
    public long id { get; set; }

    public DateTime timestamp { get; set; }

    public string? description { get; set; }

    public long level_id { get; set; }

    public long source_id { get; set; }

    public long type_id { get; set; }

    public DateTime? created_at { get; set; }

    public DateTime? updated_at { get; set; }

    public virtual level level { get; set; } = null!;

    public virtual source source { get; set; } = null!;

    public virtual type type { get; set; } = null!;
}
